tsl/test/expected/tsl_tables.out

```diff
@@ -158,23 +154,25 @@ WARNING:  could not add retention policy due to existing policy on hypertable wi
                    -1
 (1 row)
 
-SELECT * FROM _timescaledb_config.bgw_job WHERE job_type IN ('drop_chunks') ORDER BY id;
-  id  |    application_name     |  job_type   | schedule_interval | max_runtime | max_retries | retry_period |    proc_name     |      proc_schema      |       owner       | scheduled | hypertable_id |                        config                        
-------+-------------------------+-------------+-------------------+-------------+-------------+--------------+------------------+-----------------------+-------------------+-----------+---------------+------------------------------------------------------
- 1002 | Retention Policy [1002] | drop_chunks | @ 1 day           | @ 5 mins    |          -1 | @ 5 mins     | policy_retention | _timescaledb_internal | default_perm_user | t         |             1 | {"hypertable_id": 1, "retention_window": "@ 3 mons"}
-(1 row)
+SELECT * FROM _timescaledb_config.bgw_job WHERE id >= 1000 ORDER BY id;
+  id  |    application_name     | schedule_interval | max_runtime | max_retries | retry_period |    proc_name     |      proc_schema      |       owner       | scheduled | hypertable_id |                          config                           
+------+-------------------------+-------------------+-------------+-------------+--------------+------------------+-----------------------+-------------------+-----------+---------------+-----------------------------------------------------------
+ 1000 | Reorder Policy [1000]   | @ 84 hours        | @ 0         |          -1 | @ 5 mins     | policy_reorder   | _timescaledb_internal | default_perm_user | t         |             1 | {"index_name": "test_table_time_idx", "hypertable_id": 1}
```

Comment:
This job wasn't here before, so behavior seems to have changed. Is this correct/expected? Could you explain?